R/plot.R

```diff
@@ -14,3 +14,40 @@ plot.baggr <- function(x, ...) {
   l[["bg"]] <- x
   do.call(baggr_plot, l)
 }
+
+#' Set, get, and replace themes for baggr plots
+#' @rdname baggr_theme_set
+#' @importFrom ggplot2 theme_grey
+#' @importFrom bayesplot theme_default
+#' @param new New theme to use for all baggr plots
```

Comment:
must document `...` param ("A named list of theme settings."); otherwise fails R CMD CHECK

Reply:
Thanks, not sure how I missed that.